### DISCUSSION TITLE
$sortField not applying when $multiSort is true

### DISCUSSION THREAD

**Discussion by kolydart** [Power-Components/livewire-powergrid#1259] [BUG]
`public string $sortField` has no effect for default sort when `public bool $multiSort = true;`

**Comment by krjohnston** [comment]
Started to use this package recently and found this issue.
This is possible but needs to be added to the documentation.

When declaring $multiSort = true;, you need to change the sort defaults to an array property $sortArray. 

    public array $sortArray = ['name' => 'asc'];
    
    public bool $multiSort = true;
